product-features/liquidity-providers/faqs.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Common questions you might have as a liquidity provider.
+---
+
+# FAQs
+
+<details>
+
+<summary>What is the difference between swap fee and fee rebate?</summary>
+
+The **swap fee** is the total fee charged to users for executing a token swap. It's the fee that directly impacts the trader and is displayed as "Fees" in the Pool Info panel.
+
+On the other hand, the **fee rebate** is the portion of the swap fee that is distributed to liquidity providers as a reward for supplying liquidity to the pool. The remaining portion of the swap fee goes to the ALEX platform. You can also find the fee rebate percentage in the Pool Info panel.
+
+![](../../.gitbook/assets/liquidity-providers/faqs-pool-info-panel-fees.png)
+
+</details>
+
+<details>
+
+<summary>What is the swap fee percentage that goes to liquidity providers?</summary>
+
+The swap fee percentage that goes to liquidity providers is known as the **fee rebate**. It is typically set at 50%, though it can vary depending on the pool. You can check this percentage in the Pool Info panel.
```

Comment:
The swap fee percentage that goes to liquidity providers is known as the **fee rebate**. It is typically set at 50% of the swap fee, though it can vary depending on the pool. You can check this percentage in the Pool Info panel.

Reply:
Done